releasenotes/README.md

```diff
@@ -10,12 +10,41 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v48 release notes.](./v48.md)
 
+## 49.14.0 (October 8, 2020)
+
+We aren't releasing a Salesforce CLI patch this week. But we have a few announcements. 
+
+* NEW:  Want to play with the next version of Salesforce CLI before it's live? We target publicly releasing a candidate by Tuesday, although it could be anytime before Thursday, the official release day. Check [open pull requests](https://github.com/forcedotcom/cli/pulls) for early drafts of the release notes. If you find any issues or have feedback after testing, create a Github issue [here](https://github.com/forcedotcom/cli/issues) and include the release candidate version in the title. 
+
+    To install the early release candidate of the CLI to test, for example, that it works correctly with your CI jobs, run this command:
```

Comment:
feels like it's missing a word, like: ...to test, for example, "so" that it works correctly...

Reply:
yep, thx